Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -5597,12 +5599,16 @@ void AXObjectCache::updateRelations(Element& origin, const QualifiedName& attrib
     }
 
     // `commandfor` is only valid for button elements.
-    if (UNLIKELY(attribute == commandforAttr) && !is<HTMLButtonElement>(origin))
-        return;
+    if (attribute == commandforAttr) [[unlikely]] {
+        if (!is<HTMLButtonElement>(origin))
+            return;
+    }
 
     // `popovertarget` is only valid for input and button elements.
-    if (UNLIKELY(attribute == popovertargetAttr) && !is<HTMLInputElement>(origin) && !is<HTMLButtonElement>(origin))
-        return;
+    if (attribute == popovertargetAttr) [[unlikely]] {
+        if (!is<HTMLInputElement>(origin) && !is<HTMLButtonElement>(origin))
```

Comment:
It's unfortunate that this gets a lot more verbose.

Reply:
I don't have a way around this. I would say it rarely makes the code more verbose but sometimes the code has to be updated since C++ went with an annotation. However, because they went with an annotation, it works with switch statements and `else` cases, which our macros didn't support.